document/fr.opensagres.xdocreport.document/src/test/java/fr/opensagres/xdocreport/document/images/FileImageProviderTestCase.java

```diff
@@ -146,36 +146,36 @@ public void loadPNGByAndChangeResize()
         imageProvider.setResize( true );
         imageProvider.setWidth( 100f );
         Assert.assertEquals( ImageFormat.png, imageProvider.getImageFormat() );
-        Assert.assertNotNull( imageProvider.getWidth(0) );
-        Assert.assertEquals( 100f, imageProvider.getWidth(0).floatValue(), 0 );
+        Assert.assertNotNull( imageProvider.getWidth(null) );
+        Assert.assertEquals( 100f, imageProvider.getWidth(null).floatValue(), 0 );
         // Here height is computed
-        Assert.assertNotNull( imageProvider.getHeight(0) );
-        Assert.assertEquals( 91f, imageProvider.getHeight(0).floatValue(), 0 );
+        Assert.assertNotNull( imageProvider.getHeight(null) );
+        Assert.assertEquals( 91f, imageProvider.getHeight(null).floatValue(), 0 );
 
         // Use the original height
         imageProvider.setResize( false );
-        Assert.assertNotNull( imageProvider.getHeight(0) );
-        Assert.assertEquals( 200f, imageProvider.getHeight(0).floatValue(), 0 );
+        Assert.assertNotNull( imageProvider.getHeight(null) );
+        Assert.assertEquals( 200f, imageProvider.getHeight(null).floatValue(), 0 );
 
         // Use the computed height
         imageProvider.setResize( true );
-        Assert.assertNotNull( imageProvider.getHeight(0) );
-        Assert.assertEquals( 91f, imageProvider.getHeight(0).floatValue(), 0 );
+        Assert.assertNotNull( imageProvider.getHeight(null) );
+        Assert.assertEquals( 91f, imageProvider.getHeight(null).floatValue(), 0 );
 
         // Use null height
         imageProvider.setUseImageSize( false );
-        Assert.assertNull( imageProvider.getHeight(0) );
+        Assert.assertNull( imageProvider.getHeight(null) );
 
         // Use the computed height
         imageProvider.setUseImageSize( true );
-        Assert.assertNotNull( imageProvider.getHeight(0) );
-        Assert.assertEquals( 91f, imageProvider.getHeight(0).floatValue(), 0 );
+        Assert.assertNotNull( imageProvider.getHeight(null) );
+        Assert.assertEquals( 91f, imageProvider.getHeight(null).floatValue(), 0 );
 
         // Force height and compute width.
         imageProvider.setSize( null, 1000f );
-        Assert.assertNotNull( imageProvider.getWidth(0) );
-        Assert.assertEquals( 1100f, imageProvider.getWidth(0).floatValue(), 0 );
-        Assert.assertNotNull( imageProvider.getHeight(0) );
-        Assert.assertEquals( 1000f, imageProvider.getHeight(0).floatValue(), 0 );
+        Assert.assertNotNull( imageProvider.getWidth(null) );
+        Assert.assertEquals( 1100f, imageProvider.getWidth(null).floatValue(), 0 );
+        Assert.assertNotNull( imageProvider.getHeight(null) );
+        Assert.assertEquals( 1000f, imageProvider.getHeight(null).floatValue(), 0 );
```

Comment:
Could you add a test with 0.582cm like https://github.com/opensagres/xdocreport/issues/187